includes/blocks/list-block/render.php

```diff
@@ -197,8 +197,13 @@ function benenson_list_process_content( $attributes ) {
  * @param array $data Item data.
  */
 if ( ! function_exists( 'benenson_render_list_item' ) ) {
-	function benenson_render_list_item( $data ) {
-		$title = isset( $data['title'] ) ? $data['title'] : '';
+	function benenson_render_list_item( $data, $display_excerpt, $cta_text ) {
+		$title   = isset( $data['title'] ) ? $data['title'] : '';
+		$excerpt = isset( $data['excerpt'] ) ? $data['excerpt'] : '';
+
+		if ( ! empty( $display_excerpt ) && 1 === $display_excerpt ) {
```

Comment:
If this variable is given a default value in the method signature, the `empty()` check will no longer be required.